taskograph.Web/appsettings.json

```diff
@@ -1,6 +1,6 @@
 {
   "ConnectionStrings": {
-    "DefaultConnection": "Data Source=COMPUTER\\MSSQLSERVER01;Database=Taskograph;Integrated Security=True;Connect Timeout=60;Encrypt=False;Trust Server Certificate=False;Application Intent=ReadWrite;Multi Subnet Failover=False",
+    "DefaultConnection": "Data Source=DESKTOP-1KHRBVS\\SQLEXPRESS;Database=Taskograph;Integrated Security=True;Connect Timeout=60;Encrypt=False;Trust Server Certificate=False;Application Intent=ReadWrite;Multi Subnet Failover=False",
```

Comment:
Tutaj masz lokalną bazę, ale pamiętaj że w konfigach nigdy nie powinieneś podawać swoich danych tylko generyczne.

Reply:
Ups, już któryś raz z kolei mi to umknęło... Następnym razem będę pamiętać.